README.md

```
@@ -79,3 +79,23 @@ nix develop -c rainix-sol-legal
+some changes to readme
+
+some new changes
+
+some other new changes
+
+some other new changes again
+
+im trying to add more changes here again, but lets just annouce them shortly here:
+more changes to come!
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)